src/meta/processors/job/LeaderBalanceJobExecutor.cpp

```diff
@@ -286,13 +283,13 @@ folly::Future<Status> LeaderBalanceJobExecutor::executeInternal() {
 
     inLeaderBalance_ = false;
     if (failed != 0) {
-      return Status::Error("partiton failed to transfer leader");
+      LOG(INFO) << folly::stringPrintf("%d partitons failed to transfer leader", failed);
```

Comment:
Why does it still return success here?

Reply:
Because we can't expect transfer leader always succeed (it is not a definitely successful operations). Not quite related to this PR.

Reply:
ok